tracer/test/test-applications/debugger/dependency-libs/Samples.Probes.TestRuns/ProbeAttributeBase.cs

```diff
@@ -4,18 +4,22 @@ namespace Samples.Probes.TestRuns;
 
 public class ProbeAttributeBase : Attribute
 {
-    public ProbeAttributeBase(bool skip, int phase, bool unlisted, int expectedNumberOfSnapshots, string[] skipOnFrameworks)
+    public ProbeAttributeBase(bool skip, int phase, bool unlisted, int expectedNumberOfSnapshots, string[] skipOnFrameworks, bool isFullSnapshot = true, int evaluateAt = 1)
     {
         Skip = skip;
         Phase = phase;
         SkipOnFrameworks = skipOnFrameworks;
         Unlisted = unlisted;
         ExpectedNumberOfSnapshots = expectedNumberOfSnapshots;
+        IsFullSnapshot = isFullSnapshot;
+        EvaluateAt = evaluateAt;
     }
 
     public bool Skip { get; }
     public int Phase { get; }
     public string[] SkipOnFrameworks { get; }
     public bool Unlisted { get; }
     public int ExpectedNumberOfSnapshots { get; }
+    public bool IsFullSnapshot { get; set; }
```

Comment:
```suggestion
    public bool CaptureSnapshot{ get; set; }
```
To be consistent with the name we ended up choosing to use in the Probe Definition.

Reply:
Fixed in 37b994f827c41d94ef48cf656b7bf56f846b4695